doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -96,14 +96,18 @@ features:
         - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
         - BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001
-    Bluetooth LE Co-existence:
+    Bluetooth LE coexistence:
       rule: WIFI_NRF700X && NRF700X_SR_COEX
       boards_and_shields:
         - SHIELD_NRF7002EK
         - SHIELD_NRF7002EK_NRF7001
         - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
         - BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001
+    Thread coexistence:
+      rule: WIFI_NRF700X && NRF700X_THREAD_COEX
+      boards_and_shields:
+        - BOARD_NRF7002DK_NRF5340_CPUAPP
     SoftAP Mode:
```

Comment:
Can a note be added to this as a superscript with text saying `Support limited to what is essential for Wi-Fi provisioning` ?

Reply:
Updated the SoftAP mode tag as discussed